src/lib/util/EventCreator.ts

```diff
@@ -0,0 +1,29 @@
+import {OutputProcessor} from '../pmd/OutputProcessor';
+import {AsyncCreatable} from '@salesforce/kit';
+
+export class EventCreator extends AsyncCreatable {
```

Comment:
Consolidating event creation calls to this class. Will create a separate story to move out all event creation to this class so that code is easier to read.

Reply:
Agreed. Our event generation code should be universal, because it could be used in way more places than just PMD.